pyfr/solvers/base/elements.py

```diff
@@ -212,6 +212,10 @@ def set_backend(self, backend, nscalupts, nonce, linoff):
         self.scal_upts_inb = inb = backend.matrix_bank(self._scal_upts)
         self.scal_upts_outb = backend.matrix_bank(self._scal_upts)
 
+        # Allocate and bank the storage required to store the solution grads
+        # which will be used by post-processing plugins
+        self.vect_upts_inb = self._vect_upts
```

Comment:
Don't think we want to call this an `inb` (which is short for input bank, but it is not a bank per se and isn't used as an input).

Reply:
So maybe access directly `self._vect_upts` and rename `_vect_upts` to `vect_upts`?

Reply:
Removed this `_vect_upts_inb`, the implementation now accesses `_vect_upts` directly